qemu/tests/floppy.py

```diff
@@ -138,8 +138,7 @@ def test(self):
             error.context("Copying file to the floppy")
             md5_cmd = params.get("md5_cmd")
             if md5_cmd:
-                md5_source = self.session.cmd("%s %s" % (params["md5_cmd"],
-                                                         source_file))
+                md5_source = self.session.cmd("%s %s" % (md5_cmd, source_file))
```

Comment:
I think no need for the else branch here, maybe you can help update it also.
md5_source = None
if md5_cmd:
    md5_source = ****